src/Tizen.NUI/src/public/BaseComponents/ViewAccessibilityEvent.cs

```diff
@@ -0,0 +1,425 @@
+            }
+        }
+
+        internal AccessibilityDoGestureSignal GestureInfoSignal() {
+            var handle = GetControl();
+            AccessibilityDoGestureSignal ret = new AccessibilityDoGestureSignal(Interop.ControlDevel.Dali_Toolkit_DevelControl_AccessibilityDoGestureSignal(handle), false);
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+            return ret;
+        }
+
+        ///////////////////////////////////////////////////////////////////
+        // ************** AccessiblityGetDescriptionSignal ************* //
+        ///////////////////////////////////////////////////////////////////
+
+        private delegate void GetDescriptionHandlerType(IntPtr data);
+        private GetDescriptionHandlerType getDescriptionCallback;
+        private EventHandler<GetDescriptionArgs> getDescriptionHandler;
+
+        public class GetDescriptionArgs : EventArgs
+        {
+            public string description { get; internal set; }
+        }
+
+        private void OnGetDescriptionEvent(IntPtr data) {
+            if (data == IntPtr.Zero)
+                return;
+
+            var arg = new GetDescriptionArgs();
+            arg.description = StringToVoidSignal.ConvertParam1(data);
+
+            getDescriptionHandler?.Invoke(this, arg);
+        }
+
+        // This uses GetDescription signal from C++ API.
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public event EventHandler<GetDescriptionArgs> DescriptionRequested {
+            add {
+                if (getDescriptionHandler == null) {
+                    getDescriptionCallback = OnGetDescriptionEvent;
+                    GetDescriptionSignal().Connect(getDescriptionCallback);
+                }
+                getDescriptionHandler += value;
+            }
+            remove {
+                getDescriptionHandler -= value;
+                if (getDescriptionHandler == null && GetDescriptionSignal().Empty() == false)
+                    GetDescriptionSignal().Disconnect(getDescriptionCallback);
+            }
+        }
+
+        internal StringToVoidSignal GetDescriptionSignal() {
+            var handle = GetControl();
+            StringToVoidSignal ret = new StringToVoidSignal(Interop.ControlDevel.Dali_Toolkit_DevelControl_AccessibilityGetDescriptionSignal(handle), false);
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+            return ret;
+        }
+
+        ///////////////////////////////////////////////////////////////////
+        // ***************** AccessiblityGetNameSignal ***************** //
+        ///////////////////////////////////////////////////////////////////
+
+        private delegate void GetNameHandlerType(IntPtr data);
+        private GetNameHandlerType getNameCallback;
+        private EventHandler<GetNameArgs> getNameHandler;
+
+        public class GetNameArgs : EventArgs
```

Comment:
Need to add [EditorBrowsable(EditorBrowsableState.Never)] tag
and solve the build warnings here. : nest type and renaming.

Reply:
Done.